cypress/e2e/viewAnApplication.cy.ts

```diff
@@ -1,18 +1,17 @@
 import ApplicationsPage from '../pages/applications';
 import { screenPresets } from '../support/helpers';
 
-describe('Application view page', () => {
+describe('View a resident application', () => {
   screenPresets.forEach((screenPreset) => {
     it(`as a read only group user I cannot edit application details on ${screenPreset}`, () => {
-      cy.viewport(screenPreset);
-
       cy.task('clearNock');
       cy.clearCookies();
+      cy.viewport(screenPreset);
```

Comment:
Just curious, the positioning of this has shifted from near the top of the test structure to around the middle, what issue does doing this address?

Reply:
I don't really have a very scientific answer on that one. The issue was that the Nock (which is intercept for the server side props) was failing in the PL and just wouldn't hit the page. So really was a guess that it might need to do the task first to keep it happy.